### PR TITLE
Generate message using default formatter

### PR DESCRIPTION
**Pull request by hauleth**
This contains more information about why test have failed, not only raw assertion message, but left and right values as well.

### CODE REVIEW

**Comment by victorolinasc** [comment]
This looks very nice and is indeed a welcome addition. I can't review it now, but tests are failing. Could you check that?

**Comment by victorolinasc** [comment]
This is awesome @hauleth . Thank you very much. This effort is much appreciated. 

I'll take some time to review it properly and run it locally in the next 24hs as this is quite interesting for where I work too. This is essentially a one file lib so I appreciate that you didn't create a fork or another one just because of this. 

Once again, thank you very much. I hope to get back to you in the following day.

File: lib/formatter.ex [hunk 1]
```diff
@@ -59,68 +60,71 @@ defmodule JUnitFormatter do
...
-    IO.write(file, result)
-    File.close(file)
 
-    if Application.get_env(:junit_formatter, :print_report_file, false) do
-      require Logger
-      Logger.debug(fn -> "Wrote JUnit report to: #{file_name}" end)
-    end
+    :ok = File.write!(file_name, result, [:write])
+
+    IO.puts(:stderr, "Wrote JUnit report to: #{file_name}")
```
[the remark this answers is not in the document]
Comment (hauleth) [reply lib/formatter.ex]: Fixed

**Comment by victorolinasc** [comment]
Published as 3.0.0: https://hexdocs.pm/junit_formatter/readme.html

Thanks once again!

**Pull request merged**